### ISSUE TITLE
User Interface Design Ehancement

### ISSUE THREAD

**Issue by dfanaei** [tsolucio/corebos#677]
Hi Guys,
I know this is not an issue and perhaps just a luxury.
I have been using CoreBOS for a few days now and I love the functionality. 
It looks like almost everything is well-coded. 
However, the user interface look is a bit confusing as the colours and the general design is a bit too outdated (Only in my opinion. Feel free to disagree! :) ). a CRM like this that can contain lots of information on one page, Color Differences can guide the eyes to the right place and ease up the mind to feel everything is simple and easy to deal with.
Are you guys looking to improve it anytime soon?

I am not really too good with coding but I have been doing design for quite some time now.
I could offer to build some of the icons and so on just to help (in my free time) and even perhaps help with some design ideas.

**Comment by joebordes** [comment]
Thank you for the positive feedback, that always helps :-)

Our current goal is to move to [LDS](https://www.lightningdesignsystem.com/) based on web components. LDS has a very sane set of guidelines and components so we don't have to invent anything (I recommend you have a read, it is really worthwhile). We already have a bunch of [components designed](https://github.com/spikeassociates/LDSWC) and mostly looking for time to apply them to the application.

In short, there is a project for the UI change and people working on it but it is a low priority thing that gets very little time as there always seem to be more important issues to attend.

For the moment all I can say is that we have it in sight and are dedicating time to it. Hold in there!

**Comment by dfanaei** [comment]
Thanks @joebordes . It's good to know you guys are doing this.
